src/libcrun/handlers/krun.c

```diff
@@ -77,30 +95,26 @@ libkrun_exec (void *cookie, libcrun_container_t *container, const char *pathname
       if (kconf->handle_sev == NULL)
         error (EXIT_FAILURE, 0, "the container requires libkrun-sev but it's not available");
       handle = kconf->handle_sev;
+      ctx_id = kconf->ctx_id;
```

Comment:
Should it be 
```
ctx_id = kconf->ctx_id_sev;
```
?

Reply:
Maybe the suggestion is not correct but I think something is at least incorrect. Look also at line 106.

Reply:
looks like these lines should be inverted

Reply:
Yup, they were inverted, as @giuseppe pointed out. Should be fixed now.